src/utils/apolloClient/builder.ts

```diff
@@ -29,7 +29,6 @@ export default function apolloClientBuilder(getToken: () => string) {
   })
 
   return new ApolloClient({
-    ssrMode: true,
```

Comment:
How did removing the bots affect this?

Reply:
It didn't directly, the package got updated and now that parameter no longer exists

Reply:
Fair enough